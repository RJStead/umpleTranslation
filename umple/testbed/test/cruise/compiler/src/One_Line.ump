generate Java "../src-gen-umple";

class OneLine
{
    someString;

    public void foo() { someString = "Hello World"; someString = 1.0; return; }
}