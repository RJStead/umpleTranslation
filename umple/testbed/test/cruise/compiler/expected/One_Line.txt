One_Line.ump:7: error: incompatible types: double cannot be converted to String
    someString = "Hello World"; someString = 1.0; return;
                                             ^
1 error